functional-samples/tutorial.tabs-manager/README.md

```diff
@@ -0,0 +1,209 @@
+# Manage tabs
+
+This [sample](https://developer.chrome.com/docs/extensions/get-started/tutorial/popup-tabs-manager) build your first tabs manager.
+
+## Overview
+
+The guide outlines how to build a tabs manager to organize your Chrome extension and Chrome Web Store documentation tabs.
```

Comment:
In this section we try to talk about what this sample is going to teach you that other samples won't (so having a popup isn't particularly relevant, but managing tabs is). Could you take another look with that in mind?